Tests/Admin/Extension/LockExtensionTest.php

```diff
@@ -66,11 +66,18 @@ public function testConfigureFormFields()
     {
         $formMapper = $this->configureFormMapper();
         $form = $this->configureForm();
+
         $this->configureAdmin(null, null, $this->modelManager->reveal());
         $event = new FormEvent($form->reveal(), array());
 
         $this->modelManager->getLockVersion(array())->willReturn(1);
-        $form->add('_lock_version', 'hidden', array('mapped' => false, 'data' => 1))->shouldBeCalled();
+
+        // NEXT_MAJOR: remove the check and add the FQCN
+        $hiddenType = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ?
```

Comment:
A quick benchmark of 1000 calls to method_exists vs 1 call + 999 calls to a global variable should do.